skin/src/vue/c-content.vue

```diff
@@ -249,10 +251,11 @@ onUpdated(() => {
   </article>
 </template>
 <style scoped>
-.content {
-  padding: 0;
+@container xwCristal (max-width: 600px) {
+  .content {
+    padding-left: 0 var(--cr-spacing-x-small);
+  }
 }
-
 .content-loading {
```

Comment:
Our code style forces a space between selectors.
```suggestion

.content-loading {
```